src/lib/services/embed-url-parser-service/embed-url-parser-service.spec.ts

```diff
@@ -31,6 +31,30 @@ describe("EmbedUrlParserService", () => {
   ];
   const validYoutubeEmbedURLs = [`https://www.youtube.com/embed/${youtubeVideoID}`];
 
+  const mousaiPaths = {
+    album: `album/1000/artist-name/album-name`,
+    playlist: `playlist/1000/playlist-name`,
+    track: `track/1000/track-name`,
+  };
+  const createValidMousaiURLFromPath = (path: string) => [
+    `https://mousai.stream/${path}`,
+    `https://mousai.stream/${path}/embed`,
+    `http://mousai.stream/${path}`,
+    `http://mousai.stream/${path}/embed`,
+    `mousai.stream/${path}`,
+    `mousai.stream/${path}/embed`,
```

Comment:
```suggestion
```

these test cases are failing because we can't create a URL  object from it because it doesn't have `http` or `https` at the beginning. this is ok because the function that calls `isMousaiFromURL` will add `https` if it fails due to missing this scheme.